testing/scenario_app/lib/src/bogus_font_text.dart

```diff
@@ -56,9 +56,4 @@ class BogusFontText extends Scenario {
       },
     );
   }
-
-  @override
-  void onDrawFrame() {
-    // Just draw once since the content never changes.
-  }
```

Comment:
Was this removal intentional? Since it doesn't call the super implementation this empty method might be on purpose.

Reply:
It is intentional (since the super syncs a second frame then signals to take a screenshot). The no-op here was somewhat intentional too (just to write some comments on it). I don't think it'll do anything on iOS. It'll send a 'take_screenshot' channel message that'll be dropped on the floor.